fluss-server/src/main/java/com/alibaba/fluss/server/kv/KvManager.java

```diff
@@ -270,4 +281,17 @@ public KvTablet loadKv(File tabletDir) throws Exception {
         this.currentKvs.put(tableBucket, kvTablet);
         return kvTablet;
     }
+
+    public void dropRemoteKvSnapshot(PhysicalTablePath physicalTablePath, TableBucket tableBucket) {
```

Comment:
nit:
dropRemoteKvSnapshot -> deleteRemoteKvSnapshot
as we also use `delete` for log....